src/security.cs

```diff
@@ -306,7 +306,7 @@ interface SecProtocolKeys {
 	}
 
 	[Static][Internal]
-	interface KeysAccessible {
+	interface SecAccessibleKey {
```

Comment:
If we're removing `[Internal]` on `SecAttributeKey` I wonder if we should make the same change to `SecAccessibleKey`, `SecProtocolKey` and `SecAuthenticationTypeKey`. They might be useful too in the same way.